src/fidasim.f90

```diff
@@ -5142,7 +5149,7 @@ end subroutine line_basis
 subroutine plane_basis(center, redge, tedge, basis, inv_basis)
     !+ Calculates basis from 3 points on a plane with +z being the plane normal
     real(Float64), dimension(3), intent(in)              :: center
-        !+ Plane origin
+        !+ Position in cylindrical coordinates
```

Comment:
This is not necessarily has to be in cylindrical coordinates. In fact this subroutine is used in the hit_npa_detector in which beam grid coordinates are used.